buildkite/graphql/cluster_queue.graphql

```diff
@@ -0,0 +1,78 @@
+fragment ClusterQueueValues on ClusterQueue {
+    id
+    uuid
+    key
+    description             
```

Comment:
You probably want to mark this as a pointer as well so we can store it as `null` in state. And check for any others that are defined as nullable types on this ClusterQueue type (I don't think there are, but just in case)

Reply:
`description` is now a pointer - nullable in state